pages/encrypt/index.jsx

```diff
@@ -21,7 +21,7 @@ module.exports = React.createClass({
           <meta property="og:type" content="website" />
           <meta property="og:title" content="Check out this awesome video"/>
           <meta property="og:site_name" content="Mozilla Advocacy - Encryption"/>
-          <meta property="og:url" content="https://advocacy.mozilla.org/encrypt/?video=2"/>
```

Comment:
I don't think you do it here... but here instead https://github.com/mozilla/advocacy.mozilla.org/pull/103/files#diff-f3157f69c013e71be91db4856735b839L76 if you want to link to that specifically. This was meant for something else.